docs/getting-started/launching-one-node/crypto-materials.md

```diff
@@ -9,3 +9,66 @@ title: Creating Crypto Material
  SPDX-License-Identifier: CC-BY-4.0
  -->
 
+### Crypto materials configuration
+
+Hyperledger Orion uses public key cryptography from user side, for user authentication and transaction Non-Repudiation. From server side, Orion uses PKI to sign and verify all server responses to clients. 
+
+Thus, each Orion client needs certificate/key pair to sign each query and transaction he submits to server, and server has to have its own certificate/key pair to sign each response to client.
+
+From another side, each Orion server has to keep each client certificate to validate client signatures and each client has to have access to server certificate to validate servers signatures. Both server and client need access to CA certificate to validate new client/server certificates.
+
+So, to start Orion new server, each server has to have following:
+1. CA certificate (we support multiple CAs, including multilevel CAs).
+   * `caConfig.rootCACertsPath` 
+2. Its own certificate, sign by CA from above and corresponding private key.
+   * `server.identity.certificatePath` and `server.identity.keyPath`
+3. Default administrative user certificate, usually denoted as `admin` certificate.
+   * `admin.certificatePath`
+
+To work with Orion server, each client needs:
+1. CA certificate.
+```go 
+&config.ConnectionConfig{     
+   RootCAs: []string{
+      "./crypto/CA/CA.pem",
+   },
+}
```

Comment:
do we support intermediate CAs in the client SDK?

Reply:
yes, we use same `CACertCollection`as server.